onnx/defs/nn/defs.cc

```diff
@@ -3065,53 +3070,56 @@ ONNX_OPERATOR_SET_SCHEMA(
                                                                              // / 2]
 
           // Create slices of inputs to multiply with sin and cos matrices based on interleaved parameter
-          // For non-interleaved (basic) rotation, slices are created as follows,
```

Comment:
Hmmm. I believe that the `Where` condition above on `RotaryDimCond` above can be resolved at function-build time, not necessary to encode it in onnx.